modules/mobile/docs/openapi.yaml

```diff
@@ -3099,6 +3099,44 @@ paths:
       security:
         - Bearer: []
       summary: /v0/messaging/health/recipients
+  /v0/messaging/health/threads/{id}/move:
+    patch:
+      description:
+        Moves all received messages in a secure message thread to a specified folder. Sent
+        messages, draft messages, and deleted messages will not be moved.
+      operationId: post_move_thread
```

Comment:
AFAIK these operation ids are meaningless, but for consistency's sake, this should probably be called `patch_move_thread`. On a side note, I have no idea why they decided to make this a patch and not a post.